samples/README.md

```diff
@@ -1,3 +1,35 @@
 The sample pipelines give you a quick start to building and deploying machine learning pipelines with Kubeflow.
 * Follow the guide to [deploy the Kubeflow pipelines service](https://www.kubeflow.org/docs/guides/pipelines/deploy-pipelines-service/).
 * Build and deploy your pipeline [using the provided samples](https://www.kubeflow.org/docs/guides/pipelines/pipelines-samples/).
+
+
+
+
+This page tells you how to use the _basic_ sample pipelines contained in the repo.
+
+## Compile the pipeline specification
+
+Follow the guide to [building a pipeline](https://www.kubeflow.org/docs/guides/pipelines/build-pipeline/) to install the Kubeflow Pipelines SDK and compile the sample Python into a workflow specification. The specification takes the form of a YAML file compressed into a `.tar.gz` file. 
+
+For convenience, you can download a pre-compiled, compressed YAML file containing the
+specification of the `sequential.py` pipeline. This saves you the steps required
```

Comment:
```suggestion
specification of the `core/sequential/sequential.py` pipeline. This saves you the steps required
```

Reply:
Nice catch. Since the sample tests take quite some time. I'll send another PR to address it.

Reply:
Also I'll suggest that in that PR we can add some guidelines for collaborators to contribute samples together with tests.